app/views/records/_form.html.erb

```diff
@@ -12,6 +12,7 @@
         </p>
       <% end %>
       <%= render :partial => "records/edit_fields/default", :collection => f.object.terms, :as => :key, :locals => {:f => f} %>
+      <%= check_box_tag "to_review" %> <b>Check if you would like to mark this for review</b>
```

Comment:
You should wrap the text in a label so it's clearly for the "to_review" checkbox